export const getUserSubmission = async (req, res, next) => {
    try{

    }
    catch(error){
        next(error);
    }
}

export const submitSolution = async (req, res, next) => {
    try{

    }
    catch(error){
        next(error);
    }
}